src/steps/process-and-validate-plugin-options.js

```diff
@@ -29,9 +29,9 @@ const optionsProcessors = [
   {
     name: `queryDepth-is-not-a-positive-int`,
     test: ({ userPluginOptions }) =>
-      typeof userPluginOptions.schema.queryDepth !== `undefined` &&
-      (!isInteger(userPluginOptions.schema.queryDepth) ||
-        userPluginOptions.schema.queryDepth <= 0),
+      typeof userPluginOptions?.schema?.queryDepth !== `undefined` &&
```

Comment:
If you wanted to use optional chaining you can eliminate the type of undefined check i believe